system/ee/ExpressionEngine/View/addons/index.php

```diff
@@ -75,204 +133,110 @@
-                    <a class="button button--default" href="" title="Update">Update</a>
-              	  </div>
-                </div>
-              </td>
-              <td class="app-listing__cell app-listing__cell--input text--center">
-								<label class="hidden" for="tbl_6092f7c1be4f4-2-0">Select Row</label>
-								<input id="tbl_6092f7c1be4f4-2-0" class="input--no-mrg" name="selection[]" value="1" data-confirm="" type="checkbox">
-							</td>
-						</tr>
-            <tr class="app-listing__row">
-              <td>
-                <span class="collapsed-label">Name</span>
-                <a href="">
-                  <div class="addon-name-table d-flex align-items-center">
-                    <img src="http://expressionengine.test/themes/ee/asset/img/default-addon-icon.svg" alt="Add-On Name" class="addon-icon-table">
-                    <div>
-                      <strong>Add-On Name</strong><br><span class="meta-info text-muted">This is a description of the add-on.</span>
-                    </div>
-                  </div>
-                </a>
-              </td>
-              <td>
-                <span class="collapsed-label">Version</span>
-                2.1.0
-              </td>
-              <td>
-                <span class="collapsed-label">Manage</span>
-                <div class="button-toolbar toolbar">
-                  <div class="button-group button-group-xsmall">
-          	    		<a class="manual button button--default" href="" title="Manual"><span class="hidden">Manual</span></a>
-          	    		<a class="settings button button--default" href="admin.php?/cp/channels/layouts/3" title="Settings"><span class="hidden">Settings</span></a>
-              	  </div>
-                </div>
-              </td>
-              <td class="app-listing__cell app-listing__cell--input text--center">
-								<label class="hidden" for="tbl_6092f7c1be4f4-2-0">Select Row</label>
-								<input id="tbl_6092f7c1be4f4-2-0" class="input--no-mrg" name="selection[]" value="1" data-confirm="" type="checkbox">
-							</td>
-						</tr>
-            <tr class="app-listing__row">
+
+           <?php $addons = $installed; foreach ($addons as $addon):?> 
+
+            <?php if (isset($addon['update'])){
+              echo '<tr  class="app-listing__row_update">';
+            }             
+            else{
+              echo '<tr  class="app-listing__row">';
+            }?>
+              
+             
+
               <td>
                 <span class="collapsed-label">Name</span>
                 <a href="">
                   <div class="addon-name-table d-flex align-items-center">
-                    <img src="http://expressionengine.test/themes/ee/asset/img/default-addon-icon.svg" alt="Add-On Name" class="addon-icon-table">
+                    <img src="<?= $addon["icon_url"]?>" alt="Add-On Name" class="addon-icon-table">
```

Comment:
I don't think we are currently using spaces after `<?=` so let's stick to same coding style